codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
```

Comment:
I'd like to set them as informational, meaning it won't block any PRs at least for now. 

https://docs.codecov.com/docs/quick-start#tips-and-tricks